### PR TITLE
[OSDEV-802 / Django] API. Track errors that come from API users vs website.

### PR DESCRIPTION
**Pull request by VadimKovalenkoSNF**
Fix [OSDEV-802](https://opensupplyhub.atlassian.net/browse/OSDEV-802)

Update Rollbar notifications in Django app

### CODE REVIEW

**Review by killev — DISMISSED** [review]

File: src/django/api/middleware.py [hunk 1]
```diff
@@ -13,24 +12,18 @@
...
 
-
-def _report_error_to_rollbar(request, auth):
-    ROLLBAR = getattr(settings, 'ROLLBAR', {})
-    if ROLLBAR:
-        import rollbar
-        rollbar.report_exc_info(
-            sys.exc_info(),
-            extra_data={'auth': auth})
+logger = logging.getLogger(__name__)
```
Comment (roman-stolar) [inline src/django/api/middleware.py:17]: Why to we need logger here?

Comment (VadimKovalenkoSNF) [reply src/django/api/middleware.py:17]: That's my leftover. I will remove it, thank you!

**Review by roman-stolar — APPROVED** [review]
LGTM

**Review by vladsha-dev — APPROVED** [review]
LGTM

**Pull request merged**